.pre-commit-config.yaml

```
@@ -94,11 +94,10 @@ repos:
     rev: v2.2.4
     hooks:
       - id: codespell
```

```suggestion
      - id: codespell
        args: [--toml=pyproject.toml]
        additional_dependencies: [tomli]
```


This will be needed for pre-commit to work.